nbs/index.ipynb

```diff
@@ -54,8 +54,8 @@
     "    \n",
     "**StatsForecast** offers a collection of widely used univariate time series forecasting models, including exponential smoothing and automatic `ARIMA` modeling optimized for high performance using `numba`.\n",
     "\n",
-    "[Getting started](#%F0%9F%A7%AC%20Getting%20Started) •\n",
-    "[Installation](#💻-installation)\n",
+    "[Getting started](https://github.com/Nixtla/statsforecast/tree/main/#-getting-started-) •\n",
+    "[Installation](https://github.com/Nixtla/statsforecast/tree/main/#-installation)\n",
```

Comment:
Setting these as relative will allow them to work even on your branch so we can test them and (hopefully) in the docs as well.
```suggestion
    "[Getting started](#-getting-started-) •\n",
    "[Installation](#-installation)\n",
```

Reply:
But this is the source of the broken link on PyPI. Unless you somehow make them absolute links, I am not sure that problem goes away.

Reply:
Ah I thought you meant they were broken in the github README. I agree with using the full links so they work on github, the webpage and PyPI. Can you please also change the section that says `You can reproduce the results` to use the github link as well?